dds2val/src/idls/NavSatStatus.idl

```diff
@@ -0,0 +1,38 @@
+/* 
```

Comment:
Do we know where these files comes from?
I am thinking - if we know it could be an idea to add a README where we state source/version, so that we can keep track if there are newer versions.